### PR TITLE
Update troubleshoot-conditional-access.md

### PR DESCRIPTION
**Pull request by bgavrilMS**
# Pull request guidance

Thank you for your contribution. The content team is monitoring the pull requests. Once your pull request is ready for review, please add a comment such as "PR is ready for review".

### CODE REVIEW

File: support/mem/intune/device-protection/troubleshoot-conditional-access.md [hunk 1]
```diff
@@ -59,6 +59,8 @@ You can view these conditions for each device in the Azure portal and in the dev
   3. Select the *Enable Browser Access* button.
   4. In the Chrome browser, sign out of Microsoft 365 and restart Chrome.
 
+- Desktop applications must use modern authentication mechanisms, which rely on users interacting with the Identity Provider using a browser. Legacy authentication mechanisms such as apps sending the [password]([url](https://learn.microsoft.com/azure/active-directory/develop/v2-oauth-ropc)) on their own are not supported.
```
Comment (localden) [inline support/mem/intune/device-protection/troubleshoot-conditional-access.md:62]:
```suggestion
Desktop applications must use modern authentication mechanisms, which rely on users interacting with the identity provider using a web browser. Legacy authentication mechanisms such as apps [sending the password directly](https://learn.microsoft.com/azure/active-directory/develop/v2-oauth-ropc) are not supported.
```

**Review by localden — CHANGES_REQUESTED** [review]

File: support/mem/intune/device-protection/troubleshoot-conditional-access.md [hunk 2]
```diff
@@ -59,6 +59,8 @@ You can view these conditions for each device in the Azure portal and in the dev
   3. Select the *Enable Browser Access* button.
   4. In the Chrome browser, sign out of Microsoft 365 and restart Chrome.
 
+Desktop applications must use modern authentication mechanisms, which rely on users interacting with the identity provider using a web browser. Legacy authentication mechanisms such as apps [sending the password directly](https://learn.microsoft.com/azure/active-directory/develop/v2-oauth-ropc) are not supported.
+-
```
Comment (alextok) [inline support/mem/intune/device-protection/troubleshoot-conditional-access.md:63]: Do you think we need add clarity on which platform it is supported and where it is not, and in which configuration:

1. Desktop applications - must user mordern authentication.
2. Script/automation - can user MSAL.NET with Broker on Win10+.

**Review by localden — CHANGES_REQUESTED** [review]

File: support/mem/intune/device-protection/troubleshoot-conditional-access.md [hunk 3]
```diff
@@ -59,6 +59,8 @@ You can view these conditions for each device in the Azure portal and in the dev
   3. Select the *Enable Browser Access* button.
   4. In the Chrome browser, sign out of Microsoft 365 and restart Chrome.
 
+- Desktop applications must use modern authentication mechanisms, which rely on users interacting with the identity provider using a web browser or an authentication broker. Scripts and automation which rely on [sending the password directly](https://learn.microsoft.com/azure/active-directory/develop/v2-oauth-ropc) can prove device identity only on Windows 10 / Windows Server 2019 and higher, if using the authentication broker.
```
[the remark this answers is not in the document]
Comment (simonxjx) [reply support/mem/intune/device-protection/troubleshoot-conditional-access.md]: @localden, some changes are applied. Thanks.

**Comment by simonxjx** [comment]
@bgavrilMS, @localden, @alextok, changes are applied.

**Review by alextok — APPROVED** [review]
:shipit:

**Comment by simonxjx** [comment]
@bgavrilMS, @localden, @alextok, the PR is merged. Thanks.

**Pull request merged**